cont.c

```diff
@@ -1872,6 +1878,22 @@ rb_f_fiber(int argc, VALUE *argv, VALUE obj)
     return rb_f_fiber_kw(argc, argv, rb_keyword_given_p());
 }
 
+static VALUE
+rb_fiber_scheduler(VALUE klass)
+{
+    return rb_scheduler_get();
+}
+
+static VALUE
+rb_fiber_set_scheduler(VALUE klass, VALUE scheduler)
+{
+    // if (rb_scheduler_get() != Qnil) {
+    //     rb_raise(rb_eFiberError, "Scheduler is already defined!");
+    // }
```

Comment:
This can be removed, `rb_scheduler_set()` handles the previous scheduler if any.

Reply:
Ah yeah, it was an experiment - considering different semantics.

Reply:
I was considering introducing `set_scheduler` and `close_scheduler`.